src/fidesops/util/saas_util.py

```diff
@@ -171,7 +173,7 @@ def format_body(
 
 def assign_placeholders(
     value: Union[str, int], param_values: Dict[str, Any]
-) -> Optional[Union[str, int]]:
+) -> Optional[str]:
```

Comment:
I was getting a number of mypy errors because of this union type. I modified this function to always return a string to avoid having to check the type for any subsequent places where the return value might be used. This inadvertently lead to non-auth code being changed as part of this PR (offset pagination strategy).